index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Hana Sabih</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+    <!-- <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet"> -->
+  </head>
+  <body>
+    <header>
+      <h5>Welcome</h5>
+      <iconify-icon
+        icon="ic:round-menu"
+        width="24"
+        height="24"
+        class="menubar"
+      ></iconify-icon>
+    </header>
+    <section class="headline">
+      <h1>
+        Hey there. I'm Mariane<br />
+        <span> I'm a software developer</span>
+      </h1>
+      <!-- <h3>Software Developer</h3> -->
+
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Please try to follow design guidelines as much as possible by sticking to Figma design, such as colors, fonts, margins, etc.

Your design:
![image](https://user-images.githubusercontent.com/90524466/205322488-b42938a7-d70b-490f-8154-be8e752fd33e.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/205322602-54387749-700b-40ab-b3ca-b6ef8d3757f6.png)